### PR TITLE
Add bart cli instructions

### PR DESCRIPTION
**Pull request by tpmccallum**
Signed-off-by: tpmccallum tim.mccallum@fermyon.com

### CODE REVIEW

File: content/common/contributing-docs.md [hunk 1]
```diff
@@ -149,6 +149,66 @@ With all Node dependencies installed, you can now check for broken links and als
 $ npm test
 ```
 
+### 6.2 Checking Your Content - Using Bartholomew's CLI
+
+The Bartholomew Command Line Interface (CLI) Tool is called `bart`. The `bart` CLI is a tool that simplifies working with Bartholomew projects (by now you probably already know that [Bartholomew](https://www.fermyon.com/blog/introducing-bartholomew) is our in-house WebAssembly (Wasm) content management system (CMS) that powers [our official Website](https://www.fermyon.com/)). And this (our official documentation) site. The `bart` CLI is handy to ensure quality assurance of new and existing content. Installing the CLI is a cinch, so please go ahead and use it when contributing.
```
Comment (radu-matei) [inline content/common/contributing-docs.md:154]: TIL "cinch".
(https://www.merriam-webster.com/dictionary/cinch)

File: content/common/contributing-docs.md [hunk 2]
```diff
@@ -149,6 +149,66 @@ With all Node dependencies installed, you can now check for broken links and als
...
+
+To build the Bartholomew CLI from source, simply perform the following commands:
```
Comment (radu-matei) [inline content/common/contributing-docs.md:156]: I would avoid using "simply" here.

File: content/common/contributing-docs.md [hunk 3]
```diff
@@ -149,6 +149,66 @@ With all Node dependencies installed, you can now check for broken links and als
...
+FLAGS:
+    -h, --help       Prints help information
+    -V, --version    Prints version information
+
+SUBCOMMANDS:
+    calendar    Print the content calendar for a Bartholomew website
+    check       Check content and identify errors or warnings
+    help        Prints this message or the help of the given subcommand(s)
+    new         Create a new page or website from a template
+
```
Comment (radu-matei) [inline content/common/contributing-docs.md:196]: nit: empty new line here.

File: content/common/contributing-docs.md (hunk 1 above)
Comment (tpmccallum) [reply content/common/contributing-docs.md:154]: I have only traditionally know the word "cinch" as a noun which represents "a thing done with ease". TIL (thanks to your link) it is also a "a strap that holds a saddle on a horse". Gosh the English language is interesting right?

File: content/common/contributing-docs.md (hunk 3 above)
Comment (tpmccallum) [inline content/common/contributing-docs.md:196]:
```suggestion
```

**Review by tpmccallum — COMMENTED** [review]
Thanks @radu-matei, missed that extra blank line.

File: content/common/contributing-docs.md [hunk 4]
```diff
@@ -149,6 +149,65 @@ With all Node dependencies installed, you can now check for broken links and als
 $ npm test
 ```
 
+### 6.2 Checking Your Content - Using Bartholomew's CLI
+
+The Bartholomew Command Line Interface (CLI) Tool is called `bart`. The `bart` CLI is a tool that simplifies working with Bartholomew projects (by now you probably already know that [Bartholomew](https://www.fermyon.com/blog/introducing-bartholomew) is our in-house WebAssembly (Wasm) content management system (CMS) that powers [our official Website](https://www.fermyon.com/)). And this (our official documentation) site. The `bart` CLI is handy to ensure quality assurance of new and existing content. Installing the CLI is a cinch, so please go ahead and use it when contributing.
+
+To build the Bartholomew CLI from source, simply perform the following commands:
```
Comment (tpmccallum) [inline content/common/contributing-docs.md:156]:
```suggestion
To build the Bartholomew CLI from source perform the following commands:
```

**Review by tpmccallum — COMMENTED** [review]
Agreed, thanks.

File: content/common/contributing-docs.md (hunk 2 above)
Comment (tpmccallum) [reply content/common/contributing-docs.md]: Thanks, missed that one!

**Review by karthik2804 — APPROVED** [review]

**Pull request merged**